### DISCUSSION TITLE
Live Site

### DISCUSSION THREAD

**Discussion by icarusinthewings** [NovaoTech/solaris#6] [Announcements]
We are currently using Replit.com for a development host and collaboration site.
To anyone who would like to help develop: Please let me know and I will add you to the REPL.

Here is the REPL for those who just want to see the work so far:
https://replit.com/@VelocityDev/Polaris/

Pings ~ @MikeDev101

**Comment by MikeDev101** [comment]
Should we create a team for others to join?

Either way, please send me an invite to edit.
https://replit.com/@MikeDev101

**Reply by icarusinthewings** [reply]
Added

**Comment by CloudGate1234** [comment]
I only know some css and html, so I can't really help.  https://docs.turbowarp.org/development/getting-started and this https://github.com/TurboWarp/scratch-gui might help with putting the turbowarp editer on polaris.

**Comment by CloudGate1234** [comment]
also there's virtually no polaris domains. I suggest Phylos. It's the name of a galaxy and it starts with p so you'd hardly have to change the design. The project is new, so change it while you still can. otherwise you'd have to go with a sub-par domain. I also recommend using arc.io to get around 1 dollar a month which can pay for the domain. Free hosting is a thing(that doesn't scam you. free domains are not a thing).

**Reply by CloudGate1234** [reply]
getting phylos.page or phylos.dev is probably best. you can get it for 10-12 dollars a year. It's also one of the more secure domains.

**Comment by spazestudios** [comment]
I suggest adding a way for people to monetize games. also I suggest you add things like conversion to other programming languages and advanced extensions as seen in adacraft.

**Reply by icarusinthewings** [reply]
Monetization will be allowed via an external site. As I said in other posts, you can make your code closed-source. I'm not prepared to handle credit card payments.
Advanced extensions will have their own place in the extensions marketplace.
Language conversion will be a project later. I'll probably add a JavaScript converter and then also add an electron app generator.
Thanks for these suggestions 🤗
~ Velocity

**Comment by udscminidisc** [comment]
what if you want to contribute on a school computer?
